INSTALLATION.md

```diff
@@ -288,6 +289,17 @@ We have created a setup script to make configuring Talawa-API easier.
 ```
 npm run setup
 ```
+## Install Using Devpod
+This guide provides a step-by-step guide to setting up a Talawa-Api server using Devpod.
+
+Follow these steps:
+
+1. Install the Devpod GUI application or Devpod CLI. [Learn more](https://devpod.sh/docs/getting-started/install)
+2. Add a provider to Devpod. For CLI, see [here](https://devpod.sh/docs/getting-started/quickstart-devpod-cli#add-a-provider) and for the GUI app, see [here](https://devpod.sh/docs/getting-started/quickstart-vscode#add-a-provider). Use Docker which you can install from their [official documentation](https://docs.docker.com/engine/install/) or  docker compatible provider like podman, colima(better compatibility with macos).
```

Comment:
1. Be more specific with the steps. Give the full list of steps if using the CLI and the GUI, or select one or the other. Whatever the choice, the steps must be foolproof. 
2. Don't give options, make the choice for the user. We don't have tech support, to the ease of use is very important